package/kernel/linux/modules/other.mk

```diff
@@ -43,7 +43,13 @@ define KernelPackage/bluetooth
 	CONFIG_BT_HCIBTUSB_MTK=y \
 	CONFIG_BT_HCIBTUSB_RTL=y \
 	CONFIG_BT_HCIUART \
-	CONFIG_BT_HCIUART_BCM=n \
+	CONFIG_BT_HCIUART_BCM=y \
+	CONFIG_BT_HCIUART_SERDEV=y \
+	CONFIG_SERIAL_DEV_CTRL_TTYPORT=y \
+	CONFIG_SERIAL_DEV_BUS=y \
```

Comment:
The `CONFIG_SERIAL_DEV_BUS` option will be compiled into the kernel and will increase the kernel image.